lib/valkyrie/persistence/fedora/persister/orm_converter.rb

```diff
@@ -5,9 +5,10 @@ class Persister
     class OrmConverter
       attr_reader :object, :adapter
       delegate :graph, to: :object
-      def initialize(object:, adapter:)
+      def initialize(object:, adapter:, optimistic_locking_enabled: false)
```

Comment:
Can you comment as to why we need optimistic_locking_enabled here. We don't need it in other orm_converters so it might be confusing to someone down the line.